test/validation/api/crypto/test_vectors_len.h

```diff
@@ -121,6 +125,12 @@
 #define ZUC_EIA3_IV_LEN          16
 #define ZUC_EIA3_DIGEST_LEN      4
 
+/* ZUC_EIA3_256 */
+#define ZUC_EIA3_256_KEY_LEN       32
+#define ZUC_EIA3_256_IV_LEN        25
+#define ZUC_EIA3_256_DIGEST_LEN_8  8
+#define ZUC_EIA3_256_DIGEST_LEN_16 16
```

Comment:
These digest length constants are not used anywhere.

Reply:
Ack. Removed the unused macros.